composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "php": "~5.5.0|~5.6.0|~7.0.0"
+        "php": "~5.5.0|~5.6.0|>=7.0.0 <7.3.0"
```

Comment:
Are you confident the code isn't compatible with 7.3? It'd be pretty nice to push out the need for potential updates by another year!

Reply:
I'm not confident that the code is compatible with 7.3 either and this seem very much like a nice to have.

Reply:
```
git:improvement/php-7.2-and-7.1-support avatax16-lib Ω ~/.composer/vendor/bin/phpcs -p . --standard=PHPCompatibility -
-runtime-set testVersion 7.3
........................................... 43 / 43 (100%)


Time: 413ms; Memory: 12Mb
```